python/kfserving/docs/KFServingClient.md

```diff
@@ -16,11 +16,82 @@ The APIs for KFServingClient are as following:
 
 Class | Method |  Description
 ------------ | ------------- | -------------
+KFServingClient | [create_credentials](#create_credentials) | Create Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get the specified KFService|
 KFServingClient | [patch](#patch)  | Patch the specified KFService|
 KFServingClient | [delete](#delete) | Delete the specified KFService |
 
+## create_credentials
+> create_credentials(storage_type, namespace=None, **kwargs)
+
+Create or update `Secret` and `Service Account` for GCS and S3 according to credentials information. Once the `Service Account` created or updated, user can configure the `Service Account` in the [V1alpha1ModelSpec](V1alpha1ModelSpec.md) for kfserving.
```

Comment:
Grammar:

Create or update a `Secret` and `Service Account` for GCS and S3 for the provided credentials. Once the `Service Account` is applied, it may be used in the `Service Account` field of a KFService's [V1alpha1ModelSpec](V1alpha1ModelSpec.md).

Reply:
Done, thanks for the professional advice :-)